README.md

```diff
@@ -135,6 +137,11 @@ microwave.
   <p>Example of task "place cup in microwave"</p>
 </div>
 
+<div align="center">
+  <img src="docs/bowls_fp_404_compressed_gif.gif" width="294px" />
+  <p>Semi-optimal A3C agent learning on NaturalLanguagePickUpMultipleObjectTask with the object being CUPS!</p>
```

Comment:
```suggestion
  <p>A3C agent learning during training on NaturalLanguagePickUpMultipleObjectTask in one of our customized scenes with the object being CUPS!</p>
```

Reply:
Also delete the first gif. I would like to have something original from us rather than the one from ai2thor. Worst case scenario you can create it in interactive mode

Reply:
i get your point about removing the microwave gif but i also like 2 gifs
how about, if you think of a 2nd gif we can replace the microwave gif?